### PR TITLE
(feat) added ability to show radiology prices on the order basket

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

-  added ability to show radiology prices on the order basket

## Screenshots

radio

### CODE REVIEW

File: src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx [hunk 1]
```diff
@@ -205,6 +215,42 @@ export function RadiologyOrderForm({
...
+            {billableItem ? (
+              <div className={styles.listContainer}>
+                <StructuredListWrapper isCondensed>
+                  <StructuredListHead>
+                    <StructuredListRow head>
+                      <StructuredListCell head className={styles.cell}>
+                        {t("mode", "payment methods")}
+                      </StructuredListCell>
+                      <StructuredListCell head className={styles.cell}>
+                        {t("prices", "prices")}
```
Comment (makombe) [inline src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx:228]:
```suggestion
                        {t("prices", "Prices")}
```

File: src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx [hunk 2]
```diff
@@ -205,6 +215,42 @@ export function RadiologyOrderForm({
...
       >
         <div className={styles.form}>
+          <Tile id="" className={styles.prices}>
+            {billableItem ? (
+              <div className={styles.listContainer}>
+                <StructuredListWrapper isCondensed>
+                  <StructuredListHead>
+                    <StructuredListRow head>
+                      <StructuredListCell head className={styles.cell}>
+                        {t("mode", "payment methods")}
```
Comment (makombe) [inline src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx:225]:
```suggestion
                        {t("paymentMethods", "Payment methods")}
```

**Review by makombe — COMMENTED** [review]
Thanks @its-kios09 LGTM

**Review by donaldkibet — CHANGES_REQUESTED** [review]
Check the comments on `esm-procedure-app` should be done in the same manner. Since we are repeating the logic for retrieving billable services. Thanks @its-kios09

**Comment by its-kios09** [comment]
updated the code on a new PR

**Pull request closed without merging**